### ISSUE TITLE
Regex search toggle fails

### ISSUE THREAD

**Issue by hughsaunders** [t9md/atom-narrow#204]
Hey, thanks for the regex search feature, however its not working for me at the moment: 

**Reproduction Steps:**
1. Narrow Search Current Project
2. Click regex toggle

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [narrow](https://github.com/t9md/atom-narrow) package 0.41.0

### Stack Trace

Uncaught TypeError: this.toggleUseRegex is not a function

```
At /Users/hughsaunders/.atom/packages/narrow/lib/read-input.coffee:40

TypeError: this.toggleUseRegex is not a function
    at /packages/narrow/lib/read-input.coffee:40:8)
```

### Commands

```
     -0:09.8.0 core:close (div.deprecation-cop.pane-item.native-key-bindings)
     -0:06.9.0 command-palette:toggle (input.hidden-input)
     -0:05.2.0 core:confirm (input.hidden-input)
     -0:05.1.0 narrow:search-current-project (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.6 
atom-spotify2 2.1.2 
build 0.67.0 
build-gradle 0.6.0 
busy 0.7.0 
busy-signal 1.4.1 
file-type-icons 1.3.3 
file-types 0.5.5 
git-control 0.9.0 
git-diff-details 1.4.0 
git-log 0.4.1 
git-plus 7.7.0 
git-projects 1.17.0 
git-status 0.3.4 
git-time-machine 1.5.9 
goto 1.8.3 
indent-guide-improved 1.4.12 
intentions 1.1.2 
language-docker 1.1.7 
language-groovy 0.6.0 
linter 2.1.4 
linter-javac 1.9.4 
linter-pycodestyle 2.0.2 
linter-python 3.1.1 
linter-ui-default 1.2.3 
minimap 4.28.1 
minimap-bookmarks 0.4.2 
minimap-cursorline 0.2.0 
minimap-git-diff 4.3.1 
minimap-hide 0.3.0 
minimap-linter 2.0.0 
minimap-selection 4.4.0 
narrow 0.41.0 
paner 0.4.0 
project-manager 3.3.4 
simple-drag-drop-text 0.3.4 
terminal-plus 0.14.5 
tree-view-git-status 1.3.1 
underline-trailing-whitespace 1.4.2 
vim-mode-plus 0.91.0 
```

**Comment by hughsaunders** [comment]
If I enable regex search in settings rather than from the search bar, it works 👍

**Comment by t9md** [comment]
Anyway it seems to bug. 
Can you reproduce allways?

**Comment by t9md** [comment]
I checked code, its simply bug, will fix soon.

**Comment by t9md** [comment]
released v0.41.1.

**Comment by t9md** [comment]
Thanks for reporting, Im so bad to catch mouse related bug 😓
